### PR TITLE
Fix Broker plugin build in CI

### PR DESCRIPTION
**Pull request by dominiklohmann**
### :notebook_with_decorative_cover: Description

This fixes the Broker plugin setup in CI.

### :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Check if CI runs through.

### CODE REVIEW

**Review by mavam — APPROVED** [review]
Relying on the official instead of bundled CAF is something we want to do in the future anyway. (After upgrading to 0.18)

**Comment by dominiklohmann** [comment]
[ch27043]

File: nix/zeek-broker/default.nix [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "-DCAF_ROOT_DIR=${caf}"
+  ];
+
+  hardeningDisable = lib.optional isStatic "pic";
+
+  doCheck = false;
+  checkTarget = "test";
+
+  meta = with lib; {
+    description = "Zeek's Messaging Library ";
```
Comment (mavam) [inline nix/zeek-broker/default.nix:56]:
```suggestion
    description = "Zeek's Messaging Library";
```

Nit.

**Review by tobim — APPROVED** [review]
I verified that this works for both bundled as well as standalone builds of the broker plugin.
I noticed some more uninitialized variable warnings with GCC11, but that can be fixed later.

**Comment by tobim** [comment]
The static-binary build with a bundled broker plugin also doesn't work yet, we can fix that later.

**Comment by dominiklohmann** [comment]
> The static-binary build with a bundled broker plugin also doesn't work yet, we can fix that later.

I've created a story for this deficiency.

**Comment by tobim** [comment]
> > The static-binary build with a bundled broker plugin also doesn't work yet, we can fix that later.
> 
> I've created a story for this deficiency.

I already have a fix, but I don't want to delay CI again.

**Comment by dominiklohmann** [comment]
> I already have a fix, but I don't want to delay CI again.

Feel free to push again, the merge can wait a bit.

**Pull request merged**